nautobot/ui/src/utils/api.js

```diff
@@ -104,6 +115,21 @@ export const baseApi = createApi({
             },
             providesTags: ["APIData"],
         }),
+        getFieldLookupChoices: builder.query({
+            query: ({ contentType, lookupField }) => {
+                const queryParams = new URLSearchParams({
+                    content_type: contentType,
+                    field_name: lookupField,
+                    limit: 50,
+                    offset: 0,
```

Comment:
Do we need to do anything to account for filtersets with more than 50 results?

Reply:
I have no opinion here, I honestly just took the query from the old UI

Reply:
@norbert-mieczkowski-codilime if I'm not mistaken this is the lookup for available lookup choices (so `__ie`, `__e`, etc) for the given field. I think 50 is fine for now.

Reply:
Correct.